src/runtime-rs/crates/runtimes/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#[macro_use]
+extern crate slog;
+macro_rules! sl {
+    () => {
+            slog_scope::logger().new(slog::o!("subsystem" => "runtimes"))
+    };
+}
```

Comment:
I hope we can have a `meta` macro for this at some point, so that it is much easier to define a per subsystem macro rule ;)

Reply:
Thanks for your review, add macro `logger_with_subsystem`.